css/style.css

```diff
@@ -689,6 +699,42 @@ ul.chars{
     margin: 0 auto;
 }
 
+@media only screen and (max-device-width:1742px) {
```

Comment:
Properly utilize space on smaller screens. The max width may need to be adjusted, I just fiddled around with the window sizing until it looked right.

Reply:
you should probably just use a max-width query, since the design of the site is dependent on the viewport being used, not the device